modules/deploy-red-hat-openshift-container-storage.adoc

```diff
@@ -45,6 +45,6 @@
 |link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/troubleshooting_openshift_container_storage/index[Troubleshooting OpenShift Container Storage 4.5]
 
 |Migrating your {product-title} cluster from version 3 to version 4
-|link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.5/html/migration/index[Migration]
+|link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/migrating_from_ocp_3_to_4/index[Migration]
```

Comment:
(currently a 404 but seems like this will be fine once published)

Reply:
Correct